src/LetsEncrypt.Azure/McMaster.AspNetCore.LetsEncrypt.Azure.csproj

```
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
+    <PackageIcon>images/le-logo-standard.png</PackageIcon>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <DefineConstants>$(DefineConstants);FEATURE_VALIDATE_DATA_ANNOTATIONS</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.Security.KeyVault.Certificates" Version="4.0.0" />
```

Looks like the required fix for importing certs was released a few days ago. Now that this is out, let's move forward with this feature :D. I'm eager to get this merged and released.
```suggestion
    <PackageReference Include="Azure.Security.KeyVault.Certificates" Version="4.0.1" />
```

---

Cool!